### ISSUE TITLE
Outdated versions

### ISSUE THREAD

**Issue by BoasE** [JonCubed/angular2-quick-start-aspnet-core-vs2015#3]
Hi
are there plans to renews this guide to newest angular and asp core version?

forexample the angular references shouldn't be anymore RC... therefore some things like the deprecated-router references are obsolete now

**Comment by JonCubed** [comment]
Yeah I plan to update this and other repos I have, just a little time constrained at the moment. If there is an immediate need I can try looking at doing over the next week or so.
